src/index.html

```diff
@@ -4,9 +4,354 @@
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              гаджеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              планшеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              фото
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__row">
+        <div class="catalog__card card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__pic"
+          >
+          <p class="card__name">Моноблок apple A1419 iMAC 27" Retina 5K (MNED2UA/A)</p>
+          <p class="card__code">Код товара: 195434</p>
+
+          <div class="card__review">
+            <div class="stars--4 stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Отзывов: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div>Цена:</div>
+            <div class="card__price-number">69 999 грн</div>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover">
+            купить
+          </a>
+        </div>
+
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__pic"
+          >
+          <p class="card__name">Моноблок apple A1419 iMAC 27" Retina 5K (MNED2UA/A)</p>
+          <p class="card__code">Код товара: 195434</p>
+
+          <div class="card__review">
+            <div class="stars--4 stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Отзывов: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div>Цена:</div>
+            <div class="card__price-number">69 999 грн</div>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="hover">
+            купить
+          </a>
+        </div>
+
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__pic"
+          >
+          <p class="card__name">Моноблок apple A1419 iMAC 27" Retina 5K (MNED2UA/A)</p>
```

Comment:
```suggestion
          <p class="card__name">Моноблок APPLE A1419 iMac  27" Retina 5K (MNED2UA/A)</p>
```
letters case in card header don't match mockup